edb/lib/std/30-uuidfuncs.edgeql

```diff
@@ -30,6 +30,16 @@ std::uuid_generate_v1mc() -> std::uuid {
 };
 
 
+CREATE FUNCTION
+std::uuid_generate_v4() -> std::uuid {
+    CREATE ANNOTATION std::description := 'Return a version 4 UUID.';
+    SET volatility := 'Volatile';
+    USING SQL $$
+    SELECT edgedbext.uuid_generate_v4()
+    $$;
```

Comment:
```suggestion
    USING SQL FUNCTION 'edgedbext.uuid_generate_v4';
```

Reply:
OK, I'll update the v1mc function to use this too.

Reply:
I think your suggestion broke volatility tests. cc @msullivan

Reply:
It's a flake, I reran the tests.

I think the bug for those failures might actually be in postgres, but damned if I know how to prove that. I've only ever been able to repro it in CI. I was able to confirm that we are generating the correct code for those cases